### DISCUSSION TITLE
Borg checkpoint-interval 86400 is too long

### DISCUSSION THREAD

**Discussion by powerflo** [nextcloud/all-in-one#2325] [Questions] [answered]
Hi,

I migrated a lot of files (~500GB) to my Nextcloud at home and mounted a remote directory (sshfs) as a backup folder.
So the initial backup of these files takes some time and my connection was interrupted and the backup stopped.

Ok, so borg should be able to resume when restarting the backup:
https://borgbackup.readthedocs.io/en/stable/faq.html#if-a-backup-stops-mid-way-does-the-already-backed-up-data-stay-there

But in the AIO backup script the checkpoint-interval is set to 86400 (24h).
With 24h the borg feature is almost useless. Why is it set so long? 

Can the checkpoint-interval be set to a lower value? Like 1h or the default of 0,5h?

Regular backups with few changes to the files are finished faster anyway. And for (initial) backups with a lot of new files it would be very helpful to have a checkpoint.

**Comment by szaimen** [comment, marked as answer]
Hi, I am fine with using the default.

This will be done with https://github.com/nextcloud/all-in-one/pull/2331

**Reply by szaimen** [reply]
This is now released with v4.9.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
